src/libs/actions/Report.js

```diff
@@ -197,12 +197,13 @@ function getSimplifiedReportObject(report) {
  */
 function getSimplifiedIOUReport(reportData, chatReportID) {
     const transactions = _.map(reportData.transactionList, transaction => ({
-        transactionID: transaction.transactionID,
+        // TransactionID is returned from API as a String
+        transactionID: Number(transaction.transactionID),
         amount: transaction.amount,
         currency: transaction.currency,
         created: transaction.created,
         comment: transaction.comment,
-    }));
+    })).reverse(); // transactionList is returned in desc order, but we desire asc order
```

Comment:
Clarified and improved the comment.

Reply:
> `transactionList` data is returned ordered by desc creation date, they are changed to asc order because we must instead display them in the order that they were created (asc).

Reply:
Outdated, resolving this point. Please see the updated comment above.